src/service/WordService.js

```diff
@@ -1,11 +1,11 @@
 import EasyHttp from "../lib/EasyHttp";
 import Util from "../lib/Util";
 import Item from "../model/Item";
-import Progress from "../model/Progress";
+import { LOCALSTORAGE_KEY } from "../Constant";
 
 const wordUrl = "http://localhost:3000/words";
 const dateDifference = [1, 2, 3, 5, 8, 13];
-const divisionForTheNumberOfDays = 1000 * 60 * 60 * 24;
+const secondsInADay = 1000 * 60 * 60 * 24;
```

Comment:
These two constants should be moved to `Constant.js`, also use `ALL_CAPITAL_SNAKE_FORMAT` because they are global scope constant.
change the name of `dateDifference` to `MEMORIZATION_DATES` to make its meaning explicit